tests/test_kubernetes_tools.py

```diff
@@ -379,6 +379,7 @@ def test_get_container_env(self):
     def test_get_kubernetes_environment(self):
         ret = self.deployment.get_kubernetes_environment()
         assert 'PAASTA_POD_IP' in [env.name for env in ret]
+        assert 'POD_NAME' in [env.name for env in ret]
```

Comment:
Is there an integration test of sorts as well for this?

Reply:
the paasta itests are pretty slim for kubernetes. we do start a cluster and run a service I think so to some extent this is covered. We don't block on travis on it but it did work for your PR: https://travis-ci.org/Yelp/paasta/jobs/538753020

basically, don't worry about itests for now :-)